TheMovieDb/Modules/Reviews/ReviewsView.swift

```diff
@@ -1,23 +1,22 @@
 //
-//  ReviewsCollectionViewController.swift
+//  ReviewsView.swift
 //  TheMovieDb
 //
-//  Created by Javier Cueto on 31/10/21.
+//  Created by Javier Cueto on 11/11/21.
+//  
 //
 
+import Foundation
 import UIKit
 
-private let reuseIdentifier = "Cell"
+final class ReviewsView: UICollectionViewController {
 
-final class ReviewsCollectionViewController: UICollectionViewController {
-    
-    // MARK: - Properties
-    public var movie: Movie
-    private var reviews = [Review]()
+    // MARK: Properties
+    var presenter: ReviewsPresenterProtocol?
+    private var viewModel = [ReviewViewModel]() // reviews = [Review]()
     
     // MARK: - LifeCycle
-    init(with movie: Movie) {
-        self.movie = movie
+    init() {
         let layout = UICollectionViewFlowLayout()
```

Comment:
suggestion: pass the layout as a dependency with a default value, this latter allows you to change the layout dynamically

Reply:
done!